projects/packages/post-list/README.md

```diff
@@ -0,0 +1,39 @@
+# Jetpack Post List Package
+
+Enhance the classic view of the Admin section of your WordPress site.
+
+## How to use
+
+Package is published in [Packagist](https://packagist.org/packages/automattic/jetpack-post-list).
+
+Use composer to add the package to your project:
+```bash
+composer add automattic/jetpack-post-list
+```
+
+Then you need to initialize it on the `plugins_loaded` hook:
+
+```php
+function init_posts_list() {
+	Post_List::init();
+}
+
+add_action( 'init', 'init_posts_list_feature' );
```

Comment:
You mention the `plugins_loaded` hook, but the snippet below uses `init`. Which one should it be?

Reply:
I believe we actually landed on the 'admin_init' hook. 😅 I updated the README. It should be correct now. Thanks!